.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+};
+
+
```

Comment:
remove one line

Reply:
Hello Duncan, I am done to add only one line at the end. Thanks